doc/product_attribute_description.rst

```diff
@@ -0,0 +1,227 @@
+  Name                                 Description
+==============  ================================================================
+Name            Name of product
+Type            The value can be Services, Goods and Assets
+Category        Where product belongs to.
+Cost Price      This can be fixed or average. This tells how cost price should
+                be updated. THe fixed price products ahve unchanged prices and
+                average price depicts the average cost of all items in stock.
+Default UOM     Default UOM for this product.
+Active          Used to disable a product.
+Consumable      Used to make product consumable.
+Set             The product attributes can be added from here.
+Code            Specific Code given to product.
+==============  ================================================================
+
+.. _Variants:
+
+Variants
++++++++++
+
+A product variant is the Stock Keeping Unit (SKU) for the product template.
+While templates are virtual grouping of related products, the variants are
+real world products. Generally there are minor varieties in the templates.
+As a result, there are different SKUs present.
+
+For example, in the phone category, the following would be the variants:
+
+=======  ======  =========
+Variant   Color   Capacity
+=======  ======  =========
+Code 1    Black   8 GB
+Code 2    Black  16 GB
+Code 3    White   8 GB
+Code 4    White  16 GB
+=======  ======  =========
+
+The different characteristics which make up variations are known as Attributes.
+
+.. _Creating new variants:
+
+Creating Variants
++++++++++++++++++
+
+Go to ``Product``>> ``Products`` >> ``Variants``, and click on new icon
+to create a new record.
+
+.. figure:: images/20.png
+
+The variants window has the following fields:
+
+=======================  ====================================================
+ Name                                     Description
+=======================  ====================================================
+Product Template          The template of which the product is a variation of.
+Code                      A unique code for the product.
+Active                    The product can be disabled with this.
+Attributes                Attributes of the product chosen.
+=======================  ====================================================
+
+.. _Product Attributes:
+
+Product Attributes
++++++++++++++++++++
+
+The attributes are the collective characteristics of proudct variants. For
+example, in case of a phone, its attributes will be capacity, brand, color etc.
+
+.. _Attribute sets:
+
+Attribute Sets
++++++++++++++++
+
+It is a collection of attributes. These are the collective characteristics of
+variants. In case of a phone, its attributes will be brand, color, capacity etc.
+
+Attribute sets are used to associate multiple attributes for a variant as it is
+very bothersome to associate each attribute separately to separate variants.
```

Comment:
Remove this line. "Attribute sets are used to associate multiple attributes for a variant as it is
+very bothersome to associate each attribute separately to separate variants."